apps/easypid/src/features/share/slides/ShareCredentialsSlide.tsx

```diff
@@ -15,14 +15,19 @@ interface ShareCredentialsSlideProps {
   submission: FormattedSubmission
   onDecline: () => void
   isAccepting: boolean
+
+  verifierName?: string
```

Comment:
better to make this `verifierDisplay` like we have issuer display which contains name / logo etc..

Reply:
removed as this is currently empty anyway for offline flow. Will do that in the regular flow later.